core/templates/dev/head/domain/collection/CollectionRightsBackendApiService.js

```diff
@@ -41,7 +41,7 @@ oppia.factory('CollectionRightsBackendApiService', [
           }
         }, function(error) {
           if (errorCallback) {
-            errorCallback(error);
+            errorCallback(error.data);
```

Comment:
Yep, we should definitely be consistent. I just didn't change it from before, I'll make a commit changing it to errorResponse.